### PR TITLE
Applied some flake8 fixes

### PR DESCRIPTION
**Pull request by powahftw**

### CODE REVIEW

**Review by mvkv — APPROVED** [review]

File: flask_server.py [hunk 1]
```diff
@@ -61,31 +57,30 @@ def get_stats_from_log_line(log_lines):
...
+    if request.method == "POST": # User is updating settings.
+        for setting_name in request.form:
+            if len(request.form[setting_name]) == 0: continue # No update for the given setting.
+            if setting_name == "extra_ids":
+                extra_ids = list(map(int, request.form["extra_ids"].splitlines())
+                settings["extra_ids"] = extra_ids
+            else:
+                settings[setting_name] = request.form[setting_name]
         save_settings(updated_settings)
         extra_ids = updated_settings["extra_ids"]
```
Comment (mvkv) [inline flask_server.py]: Change "updated_settings" in "settings"

**Review by mvkv — COMMENTED** [review]
Remember to update store_session_id to login_and_store_session_id also in runner.py

File: flask_server.py [hunk 2]
```diff
@@ -54,83 +50,80 @@ def get_stats_from_log_line(log_lines):
...
-                    continue
-                updated_settings[setting] = request.form[setting]    
+    if "session_id" not in settings: return redirect("/login")  # Prompt the user to log-in if he's not
+    folder_path = settings.get("folder_path", DEFAULT_FOLDER)
+    extra_ids = settings.get("extra_ids", [])
+    if request.method == "POST": # User is updating settings.
+        for setting_name in request.form:
+            if len(request.form[setting_name]) == 0: continue # No update for the given setting.
+            if setting_name == "extra_ids":
+                extra_ids = list(map(int, request.form["extra_ids"].splitlines())
```
Comment (mvkv) [inline flask_server.py]: You are missing a closing bracket

File: flask_server.py (hunk 1 above)
Comment (powahftw) [reply flask_server.py]: Fixed

**Comment by powahftw** [comment]
Should be all fixed!

**Pull request merged**